### DISCUSSION TITLE
Create multiple default `statspage` pages

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5698] [Feature Requests]
**Requested By:** Dan at CSIRO

**Comment by maxpatiiuk** [comment]
@realVinayak was this implemented like requested or is it for future release?

**Comment by realVinayak** [comment]
Future release. Since the Add Stats Dialog shows all default stats not on the page, the users can make a new page and then add all default stats from the dialog. Once Add Stats Dialog supports multiple items being added (by clicking on page name or category name) this will be fixed
